### DISCUSSION TITLE
Custom column subRow for different data type? (Nested Trees, but with mixed subRow data)

### DISCUSSION THREAD

**Discussion by fullstackwebdev** [KevinVandy/material-react-table#136] [Q&A] [answered]
If I follow the "Expanding Sub Rows (Tree Data)"  example here: https://www.material-react-table.com/docs/guides/expanding-sub-rows  (Thanks)

it states: "NOTE: This feature is for expanding rows of the same data type. If you want to add expansion of more data for the same row, check out the [Detail Panel Feature Guide](https://www.material-react-table.com/docs/guides/detail-panel)."

But the Detail Panel Feature is a formatted output.

Is it possible to do another SubRow without the same data type?
For example, some way to specify `subRows` in `columns`, of a different structure, but not use Detail Panel, and use another row/table?

Reason is, my data is from GraphQL.

Thanks Kevin, great work.

**Comment by KevinVandy** [comment, marked as answer]
No, that's what Detail Panel is for. It can be in any kind of UI within the Detail panel that you want to make though.

The fact that you data comes from graphql should not matter. If you really want to join your data together into similar columns, you just need to parse it in a useMemo or something before passing it to the table.

**Reply by Spyke123** [reply]
Can you please guide me , on how to use OBS studio , I find it hard working with it

**Reply by KevinVandy** [reply]
@Spyke123 you're in the wrong github pal. This is for a table library, not OBS

**Comment by fullstackwebdev** [comment]
Thanks KevinVandy for your hard work.  .. trying to close this issue
